app/views/users/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_for @user do |u| %>
+  <p>
+    <%= u.label :username %>
+    <%= u.text_field :username, placeholder: "Your username" %>
+  </p>
+  <p>
+    <%= u.label :email %>
+    <%= u.text_field :email, placeholder: "Email" %>
+  </p>
```

Comment:
Since your expected input is an email, you should not use a text_field, instead, you should use an email_field.
Kindly check this article for more on this. [link](https://guides.rubyonrails.org/form_helpers.html)